### ISSUE TITLE
v2.2.0 doesn't work with vite

### ISSUE THREAD

**Issue by dspangenberg** [MrDeerly/vue-timeago3#245]
Hi,

the new version doesn't work with vite.

```
17:03:55 [vite] Internal server error: Failed to resolve entry for package "vue-timeago3". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "vue-timeago3". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: vite:import-analysis
  File: /Users/dspangenberg/Projects/twiceware.cloud/enclosure/src/main.js
      at packageEntryFailure (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:40578:11)
      at resolvePackageEntry (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:40574:9)
      at tryNodeResolve (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:40381:20)
      at Context.resolveId (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:40189:28)
      at async Object.resolveId (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:38754:32)
      at async TransformContext.resolve (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:38528:23)
      at async normalizeUrl (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:56682:34)
      at async TransformContext.transform (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:56837:57)
      at async Object.transform (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:38817:30)
      at async doTransform (/Users/dspangenberg/Projects/twiceware.cloud/enclosure/node_modules/vite/dist/node/chunks/dep-3397b401.js:55749:29)
```

Greetings,
Danny

**Comment by MrDeerly** [comment]
Hey Danny!

Can you give me some details about the import / utalization of the package? I‘ll also try to reproduce it soonish.

Thanks! :)

**Comment by MrDeerly** [comment]
Okay, i was able to reproduce and fix the issue. I revoked v2.2.0 and released a fixed version in v2.2.1. 

It was the same issue as in https://github.com/MrDeerly/vue-timeago3/issues/39 (The automated release workflow was still borked :/). I covered the issue in https://github.com/MrDeerly/vue-timeago3/issues/246 and will use manual releases until it's completly fixed.  

Sorry for the inconvenience!

Greets,
Chris
